src/sb_pointcloud_processing/launch/nm.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+    <!-- Toggle if using simulation data -->
+    <param name="/use_sim_time" value="true"/>
+
+    <node pkg="nodelet"
+          type="nodelet"
+          name="nm"
+          args="manager"
+          output="screen" />
+</launch>
```

Comment:
`nm` is a bit obscure. I know it's a bit longer, but could we rename this file to `nodelet_manager.launch` and rename the node itself to `nodelet_manager`? I think that would make this more readable for people coming in with no idea of how nodelets work.

Reply:
:heavy_check_mark: